src/modules/organisation/persistence/organisation.repository.ts

```diff
@@ -140,6 +142,28 @@ export class OrganisationRepository {
         return rawResult.map(mapEntityToAggregate);
     }
 
+    public async findParentOrgasForIdSorted(id: OrganisationID): Promise<Organisation<true>[]> {
```

Comment:
Noch ein JSDoc dazu wie rum sortiert wird wäre hilfreich um auch in Zukunft schnell zu wissen wie die Methode sortiert. Oder Es direkt in den Methodennamen packen

Reply:
Heißt nun findParentOrgasForIdSortedByDepthAsc